src/main/scala/com/crobox/clickhouse/ClickhouseClient.scala

```diff
@@ -4,13 +4,17 @@ import akka.NotUsed
 import akka.actor.ActorSystem
 import akka.http.scaladsl.model._
 import akka.stream.scaladsl.{Framing, Source}
-import akka.stream.{ActorMaterializer, Materializer}
 import akka.util.ByteString
-import com.crobox.clickhouse.balancing.{QueryBalancer, SingleHostQueryBalancer}
-import com.typesafe.scalalogging.LazyLogging
+import com.crobox.clickhouse.balancing.HostBalancer
+import com.crobox.clickhouse.internal.{
```

Comment:
is `.scalafmt.conf` included into the project? because it's not the default formatting